openshift/jenkins-persistent-template.json

```diff
@@ -28,7 +28,7 @@
         "tls": {
```

Comment:
I'm going to check why this is happening and post update here. This part is coming from openshift template itself so it should work fine.